plugins/native/configs/thumbnail.d/10_native.json

```diff
@@ -44,5 +44,12 @@
       "anti-aliasing": false,
       "translucency-support": false
     }
+  },
+  {
+    "match": ".*(mdl)",
+    "options": {
+      "up": "+Z",
+      "animation-index": "-1"
```

Comment:
that doesnt make much sense for thumbnail to have a animation related option tbh